services-directions-refresh-models/src/main/java/com/mapbox/api/directionsrefresh/v1/models/DirectionsRefreshJsonObject.java

```diff
@@ -0,0 +1,29 @@
+package com.mapbox.api.directionsrefresh.v1.models;
+
+import com.google.gson.GsonBuilder;
+import com.mapbox.api.directions.v5.DirectionsAdapterFactory;
+import com.mapbox.api.directions.v5.WalkingOptionsAdapterFactory;
+import com.mapbox.api.directionsrefresh.v1.DirectionsRefreshAdapterFactory;
+import com.mapbox.geojson.Point;
+import com.mapbox.geojson.PointAsCoordinatesTypeAdapter;
+import java.io.Serializable;
+
+/**
+ * Provides a base class for Directions model classes.
+ */
+public class DirectionsRefreshJsonObject implements Serializable {
+
+  /**
+   * This takes the currently defined values found inside this instance and converts it to a json
+   * string.
+   *
+   * @return a JSON string which represents this DirectionsJsonObject
+   */
+  public String toJson() {
+    GsonBuilder gson = new GsonBuilder();
+    gson.registerTypeAdapterFactory(DirectionsAdapterFactory.create());
```

Comment:
Should this be here?

Reply:
Sure, we need that because using Annotations which is a part of DirectionsRoute